ci/Jenkinsfile.android

```diff
@@ -81,7 +87,12 @@ pipeline {
     stage('Build Android APK') {
       steps {
         script {
-          linux.bundle(target='mobile-build', verbose=params.VERBOSE)
+          sh "echo STATUSQ_TESTMODE=${env.STATUSQ_TESTMODE}"
+          withEnv([
+            "STATUSQ_TESTMODE=${env.STATUSQ_TESTMODE}"
+          ]) {
+            linux.bundle(target='mobile-build', verbose=params.VERBOSE)
```

Comment:
none of these changes are needed. 
The addition of `STATUSQ_TESTMODE` in environment block should be enough.